libs/designsystem/src/lib/components/button/button.component.spec.ts

```diff
@@ -71,6 +70,11 @@ describe('ButtonComponent', () => {
       expect(element).toHaveComputedStyle({ margin: size('xxxs') });
     });
 
+    it('should have touch area with minimum size equal to fat finger size', () => {
+      expect(window.getComputedStyle(element, '::after').minHeight).toEqual(fatFingerSize());
+      expect(window.getComputedStyle(element, '::after').minWidth).toEqual(fatFingerSize());
+    });
```

Comment:
See other comment regarding test (segmented-control)

Reply:
Fixed in 44d80b33f8d0b350bdf2f3518eb6013be1bd122b